src/config.ts

```diff
@@ -616,3 +617,11 @@ export const countries: readonly CountryType[] = [
   { code: 'ZM', label: 'Zambia', phone: '260' },
   { code: 'ZW', label: 'Zimbabwe', phone: '263' }
 ];
+
+export const localStorageConfig: LocalStorageConnectorConfig = {
```

Comment:
If i understand it correclty, this config determins which type of storage is being use, right?

Maybe it is more intuitive if you can pass the type of storage you want to use when you use the `dispatch` function.

So something like this:

```  dispatch({
      storageType: "session",
      type: 'SET_SELECTED_FACILITY_ID',
      payload: facilityId
    });```
    
    We could include a defaultValue of `local` and it that we al other functionality will also still work.

Reply:
not sure. the variant in the config.ts is easier to support.
`actions` are storage-agnostic, they just work with the state, not with storage.

Reply:
As i see, so i only have to use this config if I want to save it in the `localStorage` and `sessionStorage`? And if i don't use the conifg it will use the normal 'react' state?